Documentation/kubernetes-upgrade.md

```diff
@@ -0,0 +1,75 @@
+# Upgrading Kubernetes
+
+This document describes upgrading the Kubernetes components on a cluster's master and worker nodes. For general information on Kubernetes cluster management, upgrades (including more advanced topics such as major API version upgrades) see the [Kubernetes upstream documentation](http://kubernetes.io/v1.0/docs/admin/cluster-management.html) and [version upgrade notes](http://kubernetes.io/v1.0/docs/design/versioning.html#upgrades)
```

Comment:
Noting that this doc specifically points to the v1.0 k8s docs.
Is this desirable or should we amend now to v1.1?
The http://k8s.io/v/docs/ paths don't have a URL scheme for making links to "/latest/" (that I've deciphered yet). That is inconvenient.